articles/how-to-use-local-storage-with-JavaScript/index.md

```diff
@@ -0,0 +1,284 @@
+
+- Limited storage capacity.
+
+### Main methods in Local Storage
+The primary methods when using Local Storage are `key()`, `setItem()`, `removeItem()`, `getItem()`, and `clear()`.
+
+#### key()
+This method is used to retrieve a value/string from a specific location. The index can be passed into the `key()` function as a parameter.
+
+```JavaScript
+var answer = localStorage.key(1);
+// this statement will retrieve the value of the second item in the localStorage.
+```
+
+The `key()` can also be used in a loop statement to retrieve all the items in the local storage.
+
+#### setItem()
+This function is used to store items in the local storage. An example of this function is shown below.
+
+```JavaScript
+Window.localStorage.setItem("grade","One");
+//in this case, the `grade` is the key while `One` is the value.
+```
+
+As mentioned before, we must `stringify` objects before we store them in the local storage. An example is outlined below.
+
+```JavaScript
+const Car = {
+  brand:"Suzuki",
+  color:"white",
+  price:10000
+}
+
+Window.localStorage.setItem('car', JSON.stringify(Car));
+```
+
+Failure to stringify the object will result in an error.
+
+#### getItem()
+This function is used to access or retrieve the data in the local storage. The method takes in a `key` as a parameter. It then extracts the required value from the local storage.
+
+For example, to retrieve the above `Car` object, we will use the following statement.
+
+```JavaScript
+Window.localStorage.getItem('car');
+```
+
+The above statement will return something like this;
+
+```JavaScript
+"{brand:"Suzuki",color:"white",price:"10000"}"
+```
+
+You should convert it to an object using `JSON.parse()` to use it in your code.
+
+```JavaScript
+JSON.parse(window.localStorage.getItem('car'));
+```
+
+#### removeItem()
+This method is used to delete an item from the local storage. The `removeItem()` method requires a key as a parameter.
+
+```javascript
+window.localStorage.removeItem('brand');
+```
+
+#### clear()
+This method is used to clear all values stored in the local storage. It does not require any parameters.
+
+```JavaScript
+Window.localStorage.clear()
+```
+
+### Project
+Now that we have learned about the primary functions of local storage, let's create a web application that stores, retrieves, deletes, and clears items from the local storage.
+
+Create a new folder and open it in your code editor. Create two files, `index.html` and `main.js`. The `index.html` will showcase the web page to the user, while the `main.js` will store our javascript functions. These functions will be used to access different functionalities of the `Local Storage`.
+
+### Let's Code
+Our `index.html` will have a `form` and several `buttons`, as shown below.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>localStorage</title>
+  <link rel="stylesheet" href="./css/bootstrap.css">
```

Comment:
You've added a Bootsrap link here, but you didn't instruct the readers to add a bootstrap file.

I think it's better if you provide a bootstrap cdn link here.

Reply:
Also, inform the readers that you are using bootstrap and that is causing the styling in the webpage. 

And provide a link to Bootstrap's homepage.